holidayextras/brands
# Jumbotron form refinements

#### What does this PR do? (please provide any background)

General jumbotron form styling refinements including:
* Removed extra bottom border from `.nav-tabs`.
* Added extra padding to form
* Cleared floats with `.clearfix()` instead of `overflow: hidden;`
* Refactored border-radii to display properly on mobile and on forms that don't follow tabs.
* Fixed misaligned tabs on mobile

#### What tests does this PR have?

n/a

#### How can this be tested?

Check the jumbotron form in *Pages > Hotels* in Pattern Library displays correctly on mobile and desktop. Try removing the `.nav-tabs` element form the DOM and check if the border-radii are visible on the top corners.

#### Screenshots / Screencast

##### Before
![screen shot 2016-02-09 at 16 56 31](https://cloud.githubusercontent.com/assets/2961244/12923635/622e1f82-cf4e-11e5-87b0-776de5eb6301.png)
##### After
![screen shot 2016-02-09 at 16 54 27](https://cloud.githubusercontent.com/assets/2961244/12923640/656320d0-cf4e-11e5-87ce-f28791ac8153.png)
##### Before
![screen shot 2016-02-09 at 16 55 14](https://cloud.githubusercontent.com/assets/2961244/12923655/75603f9a-cf4e-11e5-961c-178f0d99f6a0.png)
##### After
![screen shot 2016-02-09 at 16 55 04](https://cloud.githubusercontent.com/assets/2961244/12923661/7768b286-cf4e-11e5-89a4-19e643d974ba.png)

#### What gif best describes how you feel about this work?
![](http://i.giphy.com/dP9BagquCOtZS.gif)

---


#### Reviewers

**Review 1**

**Review 2** \*

**Review 3** _(optional)_

By adding a +1 you are confirming you have...
- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
- Checked for coding anti-patterns.
- Checked for appropriate test coverage.
- Checked all the tests are passing.

\*  for HX this review must be completed by an SE, SA or Project Guru
